src/Sentry.Log4Net/SentryAppender.cs

```diff
@@ -127,7 +127,7 @@ protected override void Append(LoggingEvent loggingEvent)
             _ = Hub.CaptureEvent(evt);
         }
 
-        private static IEnumerable<KeyValuePair<string, object>> GetLoggingEventProperties(LoggingEvent loggingEvent)
+        private static IEnumerable<KeyValuePair<string, object?>> GetLoggingEventProperties(LoggingEvent loggingEvent)
```

Comment:
Not sure it makes sense to make the value nullable makes sense.
We can just not add the key at all

Reply:
Decided that the change will stay.